lib/stock.js

```diff
@@ -8,8 +8,8 @@ module.exports = function (restClient) {
     }
     return false;
   }
-  module.check = (stock) => {
-    url += `check?sku=${stock.sku}&stockId=${stock.stockId}`;
+  module.check = (sku, stockId) => {
+    url += `check?sku=${sku}&stockId=${stockId}`;
     return restClient.get(url).then((data)=> {
```

Comment:
Hi @gibkigonzo ,

I made the change, is it good for you ?

Reply:
hey :) thanks for update :heart: 
can you please add one more change?
```
url += `check?sku=${sku}`;
if (stockId) {
  url += `&stockId=${stockId}`;
]
```
to prevent of creating `&stockId=undefined`

Reply:
It's good, it's done, thank you.

I still have a question.

It's been a 1 year so I can't remember exactly but to remember it's in this file

https://github.com/DivanteLtd/vue-storefront-api/blob/master/src/api/stock.js#L48

it is an object that we receive as the first parameter. 

You made a change here to make it work

https://github.com/DivanteLtd/vue-storefront-api/blob/master/src/api/stock.js#L48

Reply:
ow, you are right. Sorry for misleading. Yes, then we need to do this as you proposed
```
  module.check = (stock) => {
    url += `check?sku=${stock.sku}&stockId=${stock.stockId}`;
```

Reply:
I made the modification :)